test/addons/dlopen-ping-pong/test-worker.js

```diff
@@ -16,5 +16,5 @@ require(bindingPath);
 new Worker(`require(${JSON.stringify(bindingPath)})`, { eval: true })
   .on('error', common.mustCall((err) => {
     assert.strictEqual(err.constructor, Error);
-    assert.strictEqual(err.message, 'Module did not self-register.');
+    assert.strictEqual(err.message, `Module did not self-register: '${bindingPath}.node'.`);
```

Comment:
I guess `.node` is not required, as it is there in $bindingPath already. From the CI: https://travis-ci.com/nodejs/node/jobs/249722606#L1044-L1048

Reply:
I agree. Adding the suffix here is unnecessary

Reply:
fixed, thanks!